vignettes/teal-modules-general.Rmd

```diff
@@ -11,39 +10,41 @@ vignette: >
 
 ### Introduction
 
-`teal` is a package that extends the `shiny` framework to build interactive GUI applications using the R programming
-language. `shiny`, and hence `teal`, are implemented to allow the building of large applications by combining small,
-decoupled modules. `teal.modules.general` is a package consisting of a set of modules that are used to build `teal`
-applications. It is "general" in the sense that the intended functions of these modules are more fundamental. This is
-in contrast to the intended functions of its sister package, `teal.modules.clinical`, which is more specialized around
-clinical data. The modules of `teal.modules.general` can be combined with modules of `teal.modules.clinical` and / or
+`teal` extends the `shiny` framework, enabling the creation of interactive GUI applications using the `R`. 
+`shiny`, and `teal`facilitate the development of extensive applications through combining small, decoupled modules. 
+The `teal.modules.general` package consist of collection of modules  essential for developing `teal` applications. 
+It is "general" in the sense that the intended functions of these modules are more fundamental. This contrasts with the more specialized focus on clinical data found in the `teal.modules.clinical` package. 
+The modules from `teal.modules.general`  can be used in conjunction with modules from `teal.modules.clinical` and / or
 other `shiny` modules to build a large `teal` / `shiny` app.
 
+
 The concepts presented here require knowledge about the core features of `teal`, specifically on how to launch a `teal`
 application and how to pass data into it. Therefore, it is highly recommended to refer to the [`README`](https://insightsengineering.github.io/teal/index.html) file and 
-the introductory [vignette](https://insightsengineering.github.io/teal/articles/teal.html) of the `teal` package.
+the introductory [vignette](https://insightsengineering.github.io/teal/latest-tag/articles/teal.html) of the `teal` package.
```

Comment:
This link is broken: https://insightsengineering.github.io/teal/latest-tag/articles/teal.html

Reply:
4dea7f3ae8a19c4f27d6d1f2c73d76fcc1ea0867

Reply:
Agree, I see you already done it :-)